src/amo/reducers/addonsByAuthors.js

```diff
@@ -34,7 +35,8 @@ export const initialState: AddonsByAuthorsState = {
   loadingFor: {},
 };
 
-export const ADDONS_BY_AUTHORS_PAGE_SIZE = 6;
+export const EXTENSIONS_BY_AUTHORS_PAGE_SIZE = 10;
+export const THEMES_BY_AUTHORS_PAGE_SIZE = 12;
```

Comment:
You should use these constants in the `UserProfile` component.

Reply:
mmh but you're using the same values in the `UserProfile` component no?

Reply:
I still don't really understand why we do not use these constants in the `UserProfile` component. We are using the exact same values so far because we load/fetch exactly these numbers. I would rather use the constants here.